### DISCUSSION TITLE
Slack Integration / Provide Automatic Branch URLs

### DISCUSSION THREAD

**Discussion by hiddenest** [vercel/community#527] [Idea]
## As-is
As I think, it is good to provide [Automatic Branch URLs](https://vercel.com/docs/concepts/deployments/automatic-urls#automatic-branch-urls). in slack message directly.

For now, Vercel Slack app only gives [Automatic URLs](https://vercel.com/docs/concepts/deployments/automatic-urls#), so there is no way to get Automatic Branch URLs.  
It is a bit annoying especially while doing QA things.

## Idea
What about providing Automatic Branch URL link at the red box area? 🤔 

<img width="1111" alt="스크린샷 2022-03-30 오후 10 58 15" src="https://user-images.githubusercontent.com/8682478/160852554-e6e994b4-cbe6-4f04-901a-93ed1a9a2650.png">

Netlify, for example, the red box area is a link for Automatic Branch URLs.

![netlify-slack-deploy-notifications](https://user-images.githubusercontent.com/8682478/160854516-6aa3b51e-eb77-478c-bbb6-4786d667da7d.png)

**Comment by swarnava** [comment]
Hey there,

This sounds like a great suggestion. I have forwarded this to our Product team. :) 

Cheers!

```product/issues/415```

**Reply by hiddenest** [reply]
thanks a lot! really looking forward to it :)

**Reply by jamesarosen** [reply]
+1 another upvote for `product/issues/415`. Our QA team need a stable URL that they can use to test a given feature / bug-fix. Vercel's integrations (GitHub, Slack) all seem to generate unique URLs for each build rather than one that's stable across all builds for a branch.
